android/app/src/main/java/com/reactnativenavigation/views/RnnToolBar.java

```diff
@@ -56,6 +56,7 @@
     private DrawerLayout mDrawerLayout;
     private ActionBarDrawerToggle mDrawerToggle;
     private ArrayList<View> mMenuItems;
+    private int mButtonColor;
```

Comment:
hmm.. the next developer who would step into the code won't know that passing `null` to `setNavUpButton` implies it's called after navigating back.
Maybe if the screen is `null` - don't style the up button. Instead, we can style it in `setStyle` which is called after `StyleHelper.updateStyles` is called.
wdyt?